src/main/java/net/minecraftforge/common/ModDimension.java

```diff
@@ -37,6 +37,25 @@
  */
 public abstract class ModDimension extends ForgeRegistryEntry<ModDimension>
 {
+    /**
+     * Stops the dimension from being saved to level.dat file.
+     * Can be overridden for dynamic dimensions
+     */
```

Comment:
This documentation is backwards or at least not clear.
I'd suggest:
> Whether this dimension should be saved to the `level.dat` file.

And you should add an explanation of which effects this causes.